packages/mds-db/index.ts

```diff
@@ -89,6 +92,134 @@ async function initialize() {
   return 'postgres'
 }
 
+//TODO: break out into imported file
+function commaize(array: ReadonlyArray<string>, quote = `'`, join = ","): any {
+  return array.map((val: any) => `${stringify(val, quote)}`).join(join)
```

Comment:
The type of `val` is known to be a string, because of the type declaration of the parameter. It should not be `any` and it is unnecessary to specify a type at all.